TESTING.md

```diff
@@ -6,17 +6,19 @@ Overview
   - [Unit tests](#unit-tests)
   - [Integration tests](#integration-tests)
   - [Functional tests](#functional-tests)
+  - [Backwards Compatibility tests](#backwards-compatibility-tests)
 - [Continuous Integration](#continuous-integration)
 - [Environment misc](#environment-misc) 
 - [Misc](#misc) 
 
 # General information
-OpenSearch Dashboards uses [Jest](https://jestjs.io/) for unit and integration tests and [Selenium](https://www.selenium.dev/) for functional tests.
+OpenSearch Dashboards uses [Jest](https://jestjs.io/) for unit and integration tests, [Selenium](https://www.selenium.dev/) for functional tests and [Cypress](https://www.cypress.io/) for backwards compatibility tests.
```

Comment:
Do we need some dev docs how to setup Cypress since I believe it's not official imported into the repo yet right?